### ISSUE TITLE
Problem with build instructions

### ISSUE THREAD

**Issue by 404ClickBait** [eranif/codelite#1549]
I'm trying to follow the build instructions [here](http://codelite.org/Developers/Linux) for an ARM-based target on Ubuntu.

I get an error from CMAKE that says Could not locate GTK2. Looking in the CmakeLists.txt file I can see that this is a result of find_package(GTK2) failing to find GTK2. I think I have installed gtk according to what the build instructions say to do using the command sudo apt-get install libgtk2.0-dev.  I also manually built wxWidgets in it's own folder /home/ubuntu/wxWisdgets3.0/wxWidgets-3.0.2/

Never used cmake before and I don't understand what a "package" is. How would I [manually] locate this package on my filesystem and how do I get cmake to find it?  I have tried the bit in the instructions where it says to add "PATH=/path/to/my/other/wx/build:$PATH" to the cmake command but same results.

**Comment by eranif** [comment]
This is not really a bug... its more of "please assist me" post  (which is totally OK)
But its not the place to ask this, please use the forum: https://forums.codelite.org

**Comment by 404ClickBait** [comment]
Thank you for pointing me in the right direction.  Although not related to the program itself, I would argue that this is a bug with the build instructions.  On the CodeLite build instructions page is a link to "Support->Submit an Issue" and that link points here.

On a side note: I've now been waiting ~1 hour for the activation email from codelite.org and have not received it yet.  I have tried re-sending the activation email multiple times.  It says "the activation email has been sent to your email address".  Nothing.  I have checked and rechecked the email address, looked in my spam folder.  I am so stuck I cannot build CodeLite.
